spring-pulsar-spring-cloud-stream-binder/src/main/java/org/springframework/pulsar/spring/cloud/stream/binder/properties/PulsarBinderConfigurationProperties.java

```diff
@@ -18,7 +18,22 @@
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
+/**
+ * {@link ConfigurationProperties} for Pulsar binder configuration.
+ *
+ * @author Soby Chacko
+ */
 @ConfigurationProperties(prefix = "spring.cloud.stream.pulsar.binder")
 public class PulsarBinderConfigurationProperties {
 
+	private int partitionCount = 1;
```

Comment:
Do you mind adding a comment here (you may be waiting to do all the doc stuff at once later though).

Reply:
i will address the javadocs in one go later.